include/server/SC_WorldOptions.h

```diff
@@ -56,11 +56,7 @@ struct WorldOptions {
     const char* mNonRealTimeOutputHeaderFormat = nullptr;
     const char* mNonRealTimeOutputSampleFormat = nullptr;
 
-#if defined(_WIN32)
-    uint32 mPreferredSampleRate = 44100;
-#else
     uint32 mPreferredSampleRate = 0;
-#endif
```

Comment:
after reading your PR description i'm still unclear on whether this is actually a breaking change or not

Reply:
The problem is that it's hard to tell.

On the systems that I've tested, the default MME devices report 44100 as default sample rate, so the behavior stayed the same. 
There might be devices out there that will report different sample rate by default (their hardware sample rate ideally) which will make the default behavior different. 

This really only applies to default devices, or ASIO devices. I'll test again with ASIO to see if there's a difference in behavior.

Reply:
I tested with ASIO driver on Behringer UMC404 soundcard. Server still boots to 44100, even if the device runs at a different sample rate. This is likely driver/device dependent, but so far it looks like in most cases server boots to 44100 still.

Reply:
Thanks for investigating!